<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtSa_Dialog_MeasureSetup</class>
 <widget class="QWidget" name="RtSa_Dialog_MeasureSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*********************indoor*******************/
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
#widget
{
	background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
QLabel
{
	color:rgb(9, 73, 108);
	background-color:rgba(0,0,0,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
QPushButton
{
	border-style:flat;
	text-align:left;
	padding-left:10px;
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
QPushButton::!checked
{
	font: 13px &quot;Arial&quot;;
	font-weight:normal;
	background-color: rgba(237, 212, 0,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
QPushButton::checked
{
font: 75 13px &quot;Arial&quot;;
font-weight:bold;
background-color: rgb(255,181,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}
#RtSa_Dialog_MeasureSetup[status = &quot;indoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
/*********************outdoor*******************/
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
#widget
{
	background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
QLabel
{
	color:rgb(9, 73, 108);
	background-color:rgba(0,0,0,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
QPushButton
{
	border-style:flat;
	text-align:left;
	padding-left:10px;
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
QPushButton::!checked
{
	font: 13px &quot;Arial&quot;;
	font-weight:normal;
	background-color: rgba(237, 212, 0,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
QPushButton::checked
{
font: 75 13px &quot;Arial&quot;;
font-weight:bold;
background-color: rgb(255,181,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}
#RtSa_Dialog_MeasureSetup[status = &quot;outdoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
/*********************night*******************/
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
#widget
{
	background-color:rgba(208,182,139,0.9);
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
QLabel
{
	color:rgb(9, 73, 108);
	background-color:rgba(0,0,0,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
QPushButton
{
	border-style:flat;
	text-align:left;
	padding-left:10px;
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
QPushButton::!checked
{
	font: 13px &quot;Arial&quot;;
	font-weight:normal;
	background-color: rgba(237, 212, 0,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
QPushButton::checked
{
font: 75 13px &quot;Arial&quot;;
font-weight:bold;
background-color: rgb(255,181,0);
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
border-image: url(:/rtsa/image/night/btn_close_normal.png);
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
#pb_cancel:pressed
{
border-image: url(:/rtsa/image/night/btn_close_click.png);
}
#RtSa_Dialog_MeasureSetup[status = &quot;night&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Density">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>154</y>
      <width>234</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistent Density</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Persisten_spec">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>108</y>
      <width>234</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistent Spectogram</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Spectrum">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>62</y>
      <width>234</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistent Spectrum</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Density_Spectrogram">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>200</y>
      <width>234</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistent Density Spectrogram</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
